modules/oadp-plugins-receiving-eof-message.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-plugins-receiving-eof-message_{context}"]
+
+= Velero plugins returning "received EOF, stopping recv loop" message
+
+[NOTE]
+====
+Velero plugins are started as separate processes. Once the Velero operation has completed, either successfully or not, they exit. Therefore, if you see a `received EOF, stopping recv loop` messages in debug logs, it does not mean an error occurred, it means that a plugin operation has completed.
```

Comment:
Offering up a slight suggestion here. Take it or leave it. At the very least, "once" needs swapped out: 

Use only to mean one time. Do not use as a conjunction to mean after or when.

See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o (once)

```suggestion
Velero plugins are started as separate processes. After the Velero operation has completed, either successfully or not, they exit. Receiving a `received EOF, stopping recv loop` message in the debug logs indicates that a plugin operation has completed. It does not mean that an error has occurred. 
```